dds_glossary/services.py

```diff
@@ -256,25 +378,25 @@ def get_concept(self, concept_iri: str, lang: str = "en") -> FullConeptResponse:
             lang (str): The language. Defaults to "en".
 
         Returns:
-            FullConeptResponse: The concept with its concept schemes and relations.
+            FullConceptResponse: The concept with its concept schemes and relations.
 
         Raises:
-            HTTPException: If the concept is not found.
+            HTTPException: If the concept does not exist.
         """
         concept = get_concept(self.engine, concept_iri)
-
-        if concept:
-            return FullConeptResponse(
-                **concept.to_dict(lang=lang),
-                concept_schemes=[scheme.iri for scheme in concept.concept_schemes],
-                relations=[
-                    RelationResponse(**relation.to_dict())
-                    for relation in get_relations(self.engine, concept_iri)
-                ],
+        if concept is None:
+            raise HTTPException(
+                status_code=HTTPStatus.NOT_FOUND,
+                detail=f"Concept {concept_iri} not found.",
             )
```

Comment:
If we keep the `raise_for_empty` or whatever we'd choose to call (see two comments above), then reuse it in its generic form.

Reply:
@jsvgoncalves resolved in e137de70832c890de8938f652adb12518863378e.